src/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/topk.hpp

```diff
@@ -18,7 +18,7 @@ typedef std::tuple<
         ngraph::opset4::TopK::SortType, // sort
         InferenceEngine::Precision,     // Net precision
         InferenceEngine::Precision,     // Input precision
-        InferenceEngine::Precision,     // Output precision
+        std::vector<InferenceEngine::Precision>, // Output precision
```

Comment:
Hi @ilya-lavrenov @ilyachur. We have this draft proposal to add multiple precisions for the case of multiple output ops. Only output is changed now and it's raised for early discussion. We would like to know your suggestions on this. If this works we can add support to input as well.